pkg/hostagent/services.go

```diff
@@ -618,9 +621,7 @@ func (sep *serviceEndpoint) SetOpflexService(ofas *opflexService, as *v1.Service
 			} else {
 				sm.ServiceIp = as.Spec.ClusterIP
 			}
-			if as.Spec.SessionAffinityConfig != nil && as.Spec.SessionAffinity == "ClientIP" {
-				sm.SessionAffinity = getSessionAffinity(as.Spec.SessionAffinityConfig)
-			}
```

Comment:
and writing else here as :
sm.SessionAffinity = &opflexSessionAffinityConfig{ClientIP: opflexClientIPConfig{TimeoutSeconds: TempSessionAffinityTimer}}